src/ebonite/build/helpers.py

```diff
@@ -7,18 +7,18 @@
 from ebonite.utils.importing import module_importable
 
 
-def build_model_docker(image_name: str, model: 'core.Model', server: Server = None,
-                       image_tag='latest', force_overwrite=False, debug=False, **kwargs):
+def build_model_docker(image_params: DockerImage, model: 'core.Model', server: Server = None,
```

Comment:
same thing here (like in flask helpers)

Reply:
Fixed to support both params/name arguments (btw without kwargs, just Union argument).